generators/app/templates/gulp/tasks/report.js

```diff
@@ -0,0 +1,233 @@
+'use strict';
+
+const path = require('path');
+const through = require('through2');
+const zlib = require('zlib');
+const http = require('http');
+const fs = require('fs');
+const explore = require('source-map-explorer');
+const os = require('os');
+const opn = require('opn');
+
+const MEBIBYTE = 1024 * 1024;
```

Comment:
`MEGABYTE`? :)

Reply:
I wanted to be precise, mega while more often used is confusing.

Reply:
Got it. Should've asked Google before writing. ;) https://en.wikipedia.org/wiki/Mebibyte